content/docs/demos/statefulsets.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Configure Stateful Applications with OSM"
+description: "Secure your Kubernetes Kafka cluster with OSM"
+type: docs
+weight: 21
+---
+
+<!---
+Remember to enable PodIP localProxyMode
+-->
+This guide will illustrate how to configure stateful applications with OSM and Statefulsets in Kubernetes. For this demo, we will be installing [Apache Kafka](https://kafka.apache.org/) and its backing store [Apache Zookeeper](https://zookeeper.apache.org/) and set traffic policies allowing them to talk to one another. Finally, we'll test that we're able to produce and consume messages to/from a Kafka topic with all communication encrypted via mTLS.
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes {{< param min_k8s_version >}} or greater with a default [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) configured.
+- Have `kubectl` available to interact with the API server.
+- Have OSM version >= v1.2.0 installed.
+- Have `osm` CLI available for managing the service mesh.
+- Have [`localProxyMode`](https://deploy-preview-382--osm-docs.netlify.app/docs/api_reference/config/v1alpha2/#config.openservicemesh.io/v1alpha2.SidecarSpec) set to `PodIP` in the OSM `MeshConfig`.
+  - Most applications that run in a statefulset (Apache Kafka included) require all incoming network traffic to come via the pod IP. By default, OSM configures sends traffic over localhost, so it is important to modify that behavior via this MeshConfig setting. **The default behavior will be switched in a later version of OSM**
+
+## Demo
+
+### Install Zookeeper
```

Comment:
Is it possible to make the flow of this doc consistent with the remaining demos where the steps are clearly enumerated using a numbered list?

Reply:
I didn't see any numbered lists for the ingress or contour demos. Because of the size of the demo (and the tone of the writing), headings as a method of separating groups of steps seems to make sense